data_collection/gazette/spiders/to/to_sampaio.py

```diff
@@ -0,0 +1,57 @@
+import datetime as dt
+from urllib.parse import urlencode
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class ToSampaioSpider(BaseGazetteSpider):
+    TERRITORY_ID = "1718808"
+    name = "to_sampaio"
+    allowed_domains = ["sampaio.to.gov.br"]
```

Comment:
Conferindo a [documentação do Scrapy](https://docs.scrapy.org/en/latest/topics/spiders.html#scrapy.Spider.allowed_domains), o `allowed_domains` também permite requisições para os subdomínios. No nosso caso, a requisição para `diariooficial.sampaio.to.gov.br` também será permitida.